pkg/validation/internal/community.go

```diff
@@ -149,12 +149,20 @@ func checkMaxOpenShiftVersion(checks CommunityOperatorChecks, v1beta1MsgForResou
 
 	semVerVersionMaxOcp, err := semver.ParseTolerant(olmMaxOpenShiftVersionValue)
 	if err != nil {
-		checks.errs = append(checks.errs, fmt.Errorf("csv.Annotations.%s has an invalid value."+
+		checks.errs = append(checks.errs, fmt.Errorf("csv.Annotations.%s has an invalid value. "+
 			"Unable to parse (%s) using semver : %s",
 			olmproperties, olmMaxOpenShiftVersionValue, err))
 		return checks
 	}
 
+	truncatedMaxOcp := semver.Version{Major: semVerVersionMaxOcp.Major, Minor: semVerVersionMaxOcp.Minor}
+	if !semVerVersionMaxOcp.EQ(truncatedMaxOcp) {
```

Comment:
If `olmMaxOpenShiftVersionValue` is `4.8.0` it looks like this will not be tripped, right?

Is that okay, or do we want to do something like this:
```go
regexp.MustCompile("^v?\d+\.\d+$").MatchString(olmMaxOpenShiftVersionValue)
```